src/meetings.ts

```diff
@@ -0,0 +1,56 @@
+export interface IMeeting {
+    day: "Monday" | "Tuesday" | "Wednesday" | "Thursday" | "Friday" | "Saturday" | "Sunday";
+    room: string;
+    startTime: string;
+    endTime?: string;
+}
+
+export function formatMeeting(meeting: IMeeting): string {
+    let formatted = `**Day**: ${meeting.day}` +
+        `\n**Room**: ${meeting.room}` +
+        `\n**Starts**: ${meeting.startTime}`;
+    if (meeting.endTime) {
+        formatted += `\n**Ends**: ${meeting.endTime}`;
+    }
+    return formatted;
+}
+
+const meetings = new Map<string, IMeeting>([
```

Comment:
So how I'm currently using this is I'm importing the json file and using it as such
`schedule.Arena.starTime` How would I be able to do this to access the time and room etc etc.